lib/IRGen/GenProto.cpp

```diff
@@ -1276,7 +1280,18 @@ class AccessorConformanceInfo : public ConformanceInfo {
       }
       return *Fulfillments;
     }
-  };
+
+    void bindArchetypeAccessPathsInConformance(IRGenFunction &IGF) {
+      auto TheDecl = Conformance.getDeclContext();
```

Comment:
Also, I know this is a third rail that nobody wants to touch, but IRGen is pretty consistent about using lowercase identifiers for local variables.